packages/vscode-lit-plugin/src/test/scripts/mocha-driver.ts

```diff
@@ -0,0 +1,29 @@
+import * as path from "path";
+import Mocha from "mocha";
+import glob from "glob";
+
+/**
+ * Finds all test files and runs them.
```

Comment:
Add to the description where it's looking for tests?

Reply:
Done